### DISCUSSION TITLE
Issue with File Opening: Similar Names Causing Multiple Files to Open

### DISCUSSION THREAD

**Discussion by BeautifulContry** [sumatrapdfreader/sumatrapdf#4503] [Q&A] [answered]
To enable the "remember these settings for each document" option, make sure to turn on the "remember opened file" option. This will cause files with similar names to open together.

For example, if you only want to view the A1 file, opening A1 will also open A2 along with it.

Is there a way to fix this?

**Comment by kjk** [comment]
Please try latest [pre-release](https://www.sumatrapdfreader.org/prerelease)

**Comment by GitHubRulesOK** [comment, marked as answer]
If it is because the session is also set to remember last session and you had A1 and A2 open when closing then it will start with both those files opened that's what advanced `RestoreSession = true` is supposed to do. 

If you do not want that then set to `RestoreSession = false`
